app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
`people` is a class attribute, call it from `Person` class

Reply:
How could I find name 'self.name' and pointer to address 'self' - func outside __init__()?

Reply:
You could refer to the `Person.people` and data that is present in the `peoples` argument